src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+
+Button.propTypes = {
+  children: PropTypes.node,
+  isActive: PropTypes.bool,
+  onClick: PropTypes.func,
```

Comment:
A function needs always needs an "isRequired".

Reply:
Not sure about that.